hedera-node/docs/design/services/smart-contract-service/accounts-blocklisting.md

```diff
@@ -0,0 +1,68 @@
+
+It is possible to “squat” on an EVM address in a couple of ways:
+1. Create an account with an alias and then update the account key.
+2. Create an account with an alias and a key that’s different from the key that’s used to derive the alias from.
+
+In both cases, this results in the private key losing control over the EVM address derived from the corresponding public key.
+
+This means that there is a difference in how some situations, like accidental transfers to known EVM addresses, will be handled in Ethereum and in Hedera:
+- In Ethereum, everyone that knows the corresponding private key will be able to authorize a transaction from the known address, and the first to send a transaction would be eligible to withdraw the funds.
+- In Hedera, only the account with the associated corresponding alias would be eligible to withdraw the funds. If another key controls this account, then only that key can be used to withdraw the funds.
+
+For more information about EVM addresses, aliases and how they are used in Hedera see [Auto Account Creation](https://docs.hedera.com/hedera/core-concepts/accounts/auto-account-creation).
+
+### Example user story
+
+As a dApp developer, I have written some contracts and tests for these contracts.
+
+In Ethereum:
+- While executing tests against different environments, using some default private test keys (e.g. the ones configured with Hardhat) to create and interact with accounts, I accidentally executed my tests on Mainnet and transferred some funds to an account with public address derived from one of the default test keys.
+- As those default private test keys are well-known, anyone can use them to withdraw the funds from the account, so in effect, the funds are lost.
+
+In Hedera (current behavior):
+- While executing tests against different environments, using some default private test keys (e.g. the ones configured with Hedera Local Node) to create and interact with accounts, I accidentally executed my tests on Mainnet and transferred some funds to an account with alias derived from one of the Hedera Local Node keys.
+- As those default private test keys are well-known, anyone could have used them to create an account with the same alias but with different account admin key, leaving me with no control over the funds.
+
+In Hedera (desired behavior):
+- While executing tests against different environments, using some default private test keys (e.g. the ones configured with Hedera Local Node) to create and interact with accounts, I accidentally executed my tests on Mainnet and transferred some funds to an account with alias derived from one of the Hedera Local Node keys.
+- As those default private test keys are well-known, the ledger can be configured to own those EVM addresses and block any transfers to them, so that I cannot accidentally send funds to them.
```

Comment:
Is the inadvertent effects of this that you can't send to any of the test accounts for some of these popular products when run locally?
For production envs this is right, for local node for example this feature might have to be disabled.

Reply:
Yes, this is correct! The accounts should not be created on local environments because the test accounts with well-known keys and addresses become unusable!